logbook-api/src/main/java/org/zalando/logbook/Logbook.java

```diff
@@ -33,16 +35,35 @@ static Logbook create() {
         return builder().build();
     }
 
-    @lombok.Builder(builderClassName = "Builder")
-    static Logbook create(@Nullable final HttpLogFormatter formatter,
-            @Nullable final HttpLogWriter writer,
+    static Logbook create(
             @Nullable final Predicate<RawHttpRequest> predicate,
-            @Nullable final Obfuscator headerObfuscator,
-            @Nullable final Obfuscator parameterObfuscator,
+            @Nullable final HttpLogFormatter formatter,
+            @Nullable final HttpLogWriter writer,
+            @Nullable final HeaderObfuscator headerObfuscator,
+            @Nullable final QueryObfuscator queryObfuscator,
             @Nullable final BodyObfuscator bodyObfuscator) {
+        return Creator.create(predicate, formatter, writer, headerObfuscator, queryObfuscator, bodyObfuscator);
+    }
 
-        final LogbookFactory factory = LogbookFactory.INSTANCE;
-        return factory.create(formatter, writer, predicate, headerObfuscator, parameterObfuscator, bodyObfuscator);
+    static Creator.Builder builder() {
+        return Creator.builder();
     }
 
+    @Hack("The Lombok IDEA plugin doesn't like @Builder on static interface methods")
```

Comment:
Does anybody else have that issue?

I have the latest version of IDEA and the Lombok plugin running. It works for normal classes, but shows generated methods in interfaces as red/non-existent. Compiling/testing/running works.